framework/src/wifi_manager/wifi_manager.c

```diff
@@ -753,7 +753,7 @@ wifi_manager_result_e _wifimgr_deinit(void)
 {
 	WM_LOG_START;
 	WIFIMGR_CHECK_UTILRESULT(wifi_utils_deinit(), "[WM] wifi_utils_deinit fail", WIFI_MANAGER_FAIL);
-
+	int i = 0;
```

Comment:
I think it is better to move to 765 line.
Because, when CONFIG_ENABLE_IOTIVITY is returned as mq_close, it becomes unnecessary variable.

Reply:
Modification completed. Thanks for comment.